packages/wrangler/src/__tests__/config/findWranglerConfig.test.ts

```diff
@@ -223,7 +223,7 @@ describe("config findWranglerConfig()", () => {
 			expect(std).toEqual(NO_LOGS);
 		});
 
-		it("should error if deploy config file and user config file do not have the same base path", async () => {
+		it.skip("should error if deploy config file and user config file do not have the same base path", async () => {
```

Comment:
??

Reply:
oops, sorry I didn't meant to commit this! thanks 🙏